addons/event/static/src/client_action/event_barcode.js

```diff
@@ -116,27 +117,22 @@ export class EventScanView extends Component {
 
     onClickSelectAttendee() {
         if (this.isMultiEvent) {
-            this.actionService.doAction("event.event_registration_action");
+            this.actionService.doAction("event.event_registration_action", {
+                additionalContext: {
+                    is_from_barcode: true,
+                },
+            });
         } else {
             this.actionService.doAction("event.event_registration_action_kanban", {
                 additionalContext: {
                     active_id: this.eventId,
                     search_default_unconfirmed: true,
                     search_default_confirmed: true,
+                    is_from_barcode: true,
```

Comment:
Ah yes tiny thing but probably to apply where possible in the PR: let us keep the alphabetical order, much easier to maintain (in dicts, keys...) thanks!